libraries/chain/proposal_object.cpp

```diff
@@ -33,15 +34,17 @@ bool proposal_object::is_authorized_to_execute(database& db) const
 
    try {
       bool allow_non_immediate_owner = ( db.head_block_time() >= HARDFORK_CORE_584_TIME );
-      verify_authority( proposed_transaction.operations, 
-                        available_key_approvals,
-                        [&]( account_id_type id ){ return &id(db).active; },
-                        [&]( account_id_type id ){ return &id(db).owner;  },
-                        allow_non_immediate_owner,
-                        db.get_global_properties().parameters.max_authority_depth,
-                        true, /* allow committeee */
-                        available_active_approvals,
-                        available_owner_approvals );
+
+      verify_authority(proposed_transaction.operations,
+                       available_key_approvals,
+                       [&]( account_id_type id ){ return &id(db).active; },
+                       [&]( account_id_type id ){ return &id(db).owner;  },
```

Comment:
Capture explicitly please

Reply:
Since I'm already changing these lines, can I fix the padded parentheses? I note that the use of padding on the insides of parentheses is *not* consistently applied throughout the code, and while I proudly accept a great deal of responsibility for that, I would point out that even Dan was not consistent in it in his own original commits.

I'll promise not to change it on lines I'm not already rewriting. =)

Reply:
I guess "padded parentheses" means the whitespace after `(` and before `)`? Actually if you ask me, I'd say please revert ALL such changes done in this PR (leave the whitespaces there). When I was rewriting code, I always add whitespaces **for consistence** (at least I think it is). 
Whitespaces are helpful when navigating in VIM with key `SHIFT + w`. xD

Reply:
Anyway, let's move the discussion about (controversial) coding style to #1318. Explicitly capturing is not controversial IMHO.